joblib/parallel.py

```diff
@@ -1588,7 +1588,7 @@ def print_progress(self):
                 is_last_item = (index + 1 == total_tasks)
                 if (is_last_item or cursor % frequency):
                     return
-            remaining_time = (elapsed_time / index) * \
+            remaining_time = (elapsed_time / max(index, 1)) * \
                              (self.n_dispatched_tasks - index * 1.0)
```

Comment:
I'm also not sure why this` * 1.0` is necessary, it seems like `index` should always be an `int` so should get automatically upcast as needed. But maybe at some point someone got burned by a `uint` subtraction problem, divide not meaning true_divide, or something?

Reply:
This seems to originate from PR #330, not sure why it is the case. It is possible that this was only to make sure we don't change the behavior but I would say it is fine to remove it.
```suggestion
                             (self.n_dispatched_tasks - index)
```